apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -785,6 +770,47 @@ class ScriptEditor extends React.Component {
           </div>
         </CollapsibleEditorSection>
 
+        <CollapsibleEditorSection title="Unit Calendar Settings">
+          <label>
+            Show Calendar
+            <input
+              type="checkbox"
+              checked={this.state.showCalendar}
+              style={styles.checkbox}
+              onChange={() =>
+                this.setState({showCalendar: !this.state.showCalendar})
+              }
+            />
+            <HelpTip>
+              <p>
+                Check to enable the calendar view on the Unit Overview Page. The
+                calendar displays each lesson and generally how long it will
+                take as well how many weeks the unit is expected to take in
+                general. (Actual calendar UI coming soon!)
```

Comment:
We will want to update this text once the calendar is live. (I wrote this a while ago)